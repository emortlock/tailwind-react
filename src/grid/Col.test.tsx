import React from 'react'
import { render, RenderResult } from '@testing-library/react'

import { RawCol as Col } from '.'

const setup = (
  testProps = {},
): [RenderResult, { props: Record<string, any> }] => {
  const props = Object.assign({}, testProps)

  const wrapper = render(<Col {...props}>Content</Col>)

  return [
    wrapper,
    {
      props,
    },
  ]
}

describe('Col', () => {
  it('renders matching snapshot', () => {
    const [{ asFragment }] = setup({
      size: { def: 'full', sm: '1/2', md: '1/4' },
    })

    expect(asFragment()).toMatchSnapshot()
  })
})
