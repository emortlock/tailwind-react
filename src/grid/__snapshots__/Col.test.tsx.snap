// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col renders matching snapshot 1`] = `
<DocumentFragment>
  <li
    class="w-full"
  >
    Content
  </li>
</DocumentFragment>
`;
